website/docs/docs/collaborate/column-level-lineage.md

```diff
@@ -54,3 +58,17 @@ Possible error cases are:
 - **Parsing error** &mdash; Error occurs when the SQL is ambiguous or too complex for parsing. An example of ambiguous parsing scenarios are _complex_ lateral joins.
 - **Python error** &mdash; Error occurs when a Python model is used within the lineage. Due to the nature of Python models, it's not possible to parse and determine the lineage.
 - **Unknown error** &mdash; Error occurs when the lineage can't be determined for an unknown reason. An example of this would be if a dbt best practice is not being followed, like using hardcoded table names instead of `ref` statements.
+
+### Data platform support
+
+CLL in dbt Cloud works with the following data platforms: 
+- Snowflake
+- BigQuery
+- Redshift
+- Databricks (Unity Catalog) 
+
+The following adapters aren't currently supported by CLL in dbt Cloud. More of these platforms will be supported in the future. 
```

Comment:
sugg active voice, non blocker
```suggestion
Currently, dbt Cloud's CLL doesn't support the following adapters. It will support more of these platforms in the future:
```